src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+    background: url(../images/star.svg) no-repeat center;
+  }
+
+  &__star:nth-child(-n + 4){
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
You should reuse logic from task stars.
If I add class stars--2 I should see only 2 active stars
The same with other modifiers